apps/web/app/landing-page/Hero.tsx

```diff
@@ -38,16 +40,15 @@ export const Hero = () => {
 
   const { toast } = useToast();
 
-  useEffect(() => {
-    (async () => {
-      const response = await fetch(ApiRoutes.Subscribers)
-      if (response.ok) {
-        const count = await response.json()
-        setSubscribersCount(count)
-      }
-    })()
-  }, [])
+  const getSubscribersCount = async () => {
+    const response = await fetch(ApiRoutes.Subscribers)
+    if (response.ok) {
+      const count = await response.json()
+      return count
+    }
+  }
```

Comment:
Please add type and return something.

```suggestion
  const getSubscribersCount = async (): Promise<number | null> => {
    const response = await fetch(ApiRoutes.Subscribers)
    if (!response?.ok) return null
    const count = await response.json()
    return count
  }
```